src/generator/ssg.integration.js

```diff
@@ -35,24 +35,6 @@ test('generates a simple playbook with multiple compliance remediation', async (
     expect(normalizePlaybookVersionForSnapshot(res.text)).toMatchSnapshot();
 });
 
-test('ignores rsyslog_remote_loghost rules for compliance remediations', async () => {
-    const data = {
-        issues: [{
-            id: 'ssg:rhel7|pci-dss|xccdf_org.ssgproject.content_rule_disable_prelink',
-            systems: ['68799a02-8be9-11e8-9eb6-529269fb1459']
-        }, {
-            id: 'ssg:rhel7|standard|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost',
-            systems: ['68799a02-8be9-11e8-9eb6-529269fb1459']
-        }]
-    };
-
-    const res = await request
-    .post('/v1/playbook')
-    .send(data)
-    .expect(200);
```

Comment:
I guess this test could stay, just expect a 400 here.

Reply:
@akofink I replaced the test with one slightly lower down, it now looks for the correct error message.

"UNSUPPORTED_ISSUE"